python/cudf/cudf/tests/test_datetime.py

```diff
@@ -1299,6 +1300,24 @@ def test_is_leap_year():
     assert_eq(expect2, got2)
 
 
+@pytest.mark.parametrize("dtype", DATETIME_TYPES)
+def test_days_in_months(dtype):
+    nrows = 10000
```

Comment:
```suggestion
    nrows = 1000
```
can we have lower number of rows, since we are only validating for the correctness of the results that is a random input?